mantidimaging/gui/test/gui_system_loading_test.py

```diff
@@ -108,11 +108,11 @@ def test_load_log(self, mocked_select_file):
 
         self.assertEqual(len(self.main_window.presenter.datasets), 1)
         sample = list(self.main_window.presenter.datasets)[0].sample
-        self.assertNotIn("log_file", sample.metadata)
+        self.assertIn("log_file", sample.metadata)
 
-        # Initial angles are just evenly spaced from 0 to 2*pi
-        stack_len = sample.num_images
-        self.assertAlmostEqual(sample.projection_angles().value[1], 2 * math.pi / (stack_len - 1), 12)
+        # Assert initial angles are the same as log_file angles
+        log_angle = math.radians(self._get_log_angle(log_path))
+        self.assertAlmostEqual(sample.projection_angles().value[1], log_angle, 12)
```

Comment:
I think it might work better here to remove the existing log, so that we can check that `actionSampleLoadLog` does something. Otherwise the test still passes if I comment out lines 118 to 120.

I'd roll back the changes, and instead put after line 110
```
        sample.log_file = None
        sample._projection_angles = None
```

And the same in `test_load_angles()`